### DISCUSSION TITLE
Support for static className in classNames

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#4231] [Feature requests]
Hi.
I would like to add some html className(s) for the internal elements of a particular component.
It is very useful if you use tailwind.
As I know from the docs, currently only selectors from `createStyles` supported and named as `classNames`, so for the cases when I want just to add a little className to internal part of a component - there is no choice.
Of course each part of component has also the static className, but this is not the case.

So the idea is to add support in classNames to pass a static selector, and internally to check if it is not from createStyles result - to add it statically. PS: I may be talking nonsense because I haven't looked at the insides of how this is set up now.

**Comment by rtivital** [comment]
I do not really understand, you can add static classes with Styles API the same way as with createStyles

**Comment by 7iomka** [comment]
> I do not really understand, you can add static classes with Styles API the same way as with createStyles

Hm
For some reason for some components this didn't worked for me
I will have to find out which components it didn't work with, or if it was an obscure glitch. 
I apologize for the inconvenience)
